### ISSUE TITLE
Missing Discriminator Weights for pretraining

### ISSUE THREAD

**Issue by sushant097** [TAMU-VITA/DeblurGANv2#155]
I try to fine-tune the available InceptionResNet-v2 weights. But only generator weights are available. Is there a way you can provide the full generator and discriminator weights, which can be directly used for fine-tuning in another dataset? It would be useful for all of us.

Thank you.

![image](https://github.com/VITA-Group/DeblurGANv2/assets/30827903/66055e65-0303-4189-a4a4-b3227b049db7)

**Comment by Lyn4444** [comment]
your loss of adv gan is rise?
